javascript/node/selenium-webdriver/io/exec.js

```diff
@@ -115,7 +115,8 @@ module.exports = function exec(command, opt_options) {
 
   var proc = childProcess.spawn(command, options.args || [], {
     env: options.env || process.env,
-    stdio: options.stdio || 'ignore'
+    stdio: options.stdio || 'ignore',
+    detached: true
```

Comment:
You're welcome to surface this option through the API, but I'm not going to make this the default behavior.